spec/namespaces/indices.yaml

```diff
@@ -2455,6 +2470,9 @@ components:
             type: object
             additionalProperties:
               $ref: '../schemas/indices.get_alias.yaml#/components/schemas/IndexAliases'
+    indices.get_alias@404:
+      content:
+        application/json: {}
```

Comment:
since the schema for `application/json` is not defined, does that mean we don't care what it is, or it's an empty JSON, or a null for body?

Reply:
There's actually a schema, but I am deferring this to https://github.com/opensearch-project/opensearch-api-specification/issues/445 if that's ok.